build_docs.py

```diff
@@ -49,7 +49,8 @@
 LANGUAGES = [
     'en',
     'fr',
-    'ja'
+    'ja',
+    'ko'
```

Comment:
should we just add a trailing `,` awaiting for future additions :-D

Reply:
I like trailing comas for auto-generated code, not for human written code. (I agree it would add diff readability, still don't really like them for humans). We're having a big list of future additions (https://mdk.fr/blog/python-documentation-translation.html) :-D